src/sha256.h

```diff
@@ -3,7 +3,7 @@
 * Author:     Brad Conte (brad AT bradconte.com)
 * Copyright:
 * Disclaimer: This code is presented "as is" without any guarantees.
-* Details:    Defines the API for the corresponding SHA1 implementation.
+* Details:    Defines the API for the corresponding SHA256 implementation.
```

Comment:
Fun fact, this is also wrong in the original code that Salvatore got this from :D